### PR TITLE
Update _d585.urdf.xacro

### PR DESCRIPTION
**Pull request by SamerKhshiboun**
Updates after new measurements of new model on solid work. Still need to be updated for every new engineering model.

### CODE REVIEW

File: realsense2_description/urdf/_d585.urdf.xacro [hunk 1]
```diff
@@ -47,11 +47,11 @@ aluminum peripherial evaluation case.
     <xacro:property name="d585_cam_depth" value="0.0931"/>
 
     <!-- TODO Samer: This parameter is still missing, took it from D455 -->
-    <xacro:property name="d585_cam_mount_from_center_offset" value="0.0377"/> <!-- TODO Samer: check -->
+    <xacro:property name="d585_cam_mount_from_center_offset" value="0.042"/> <!-- TODO Samer: check -->
```
Comment (Nir-Az) [inline realsense2_description/urdf/_d585.urdf.xacro]: TODO Samer: check?

Comment (Nir-Az) [reply realsense2_description/urdf/_d585.urdf.xacro]: Same for other lines

Comment (SamerKhshiboun) [reply realsense2_description/urdf/_d585.urdf.xacro]: We decided to keep these lines as long as we don't have the accurate numbers.

Comment (Nir-Az) [reply realsense2_description/urdf/_d585.urdf.xacro]: I suggest comment this :)
```Not final numbers, still require refinements```
TODO Samer check means you are not sure and you need to check before it is merged :)

Comment (SamerKhshiboun) [reply realsense2_description/urdf/_d585.urdf.xacro]: fixed

**Review by Nir-Az — APPROVED** [review]

**Pull request merged**